frontend/routes/package/(_components)/PackageHeader.tsx

```diff
@@ -82,7 +82,7 @@ export function PackageHeader(
                 <span>
                   <a
                     href={`/@${pkg.scope}`}
-                    class="link font-bold no-underline"
+                    class="link font-bold pr-1 no-underline"
```

Comment:
Not color related but may help with an issue of readability people have expressed where they didn't necessarily know the scope and package were distinct links here